modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -0,0 +1,94 @@
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-configuring-distr-tracing-tempo_{context}"]
+= Configuring the distributed tracing platform (Tempo)
+
+You can expose tracing data to the {TempoName} stack by appending a named element and `zipkin` provider to the `spec.meshConfig.extensionProviders` specification. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the Tempo distributor service endpoint.
+
+The following example configures the `ServiceMeshControlPlane` resource to define an extensionProvider `zipkin` for sending tracing spans to a TempoStack. You can create a TempoStack instance in the `tracing-system` namespace _after_ creating the `ServiceMeshControlPlane` and the `ServiceMeshMemberRoll` resources.
+
+.Prerequisites
+
+You have installed the {TempoOperator} and {SMProductName} Operator in the `openshift-operators` namespace and have created namespaces such as `istio-system` and `tracing-system`.
+
+.Procedure
+
+. Configure the `ServiceMeshControlPlane` resource to define an extension provider:
++
+[source,yaml]
+----
+kind: ServiceMeshControlPlane
+apiVersion: maistra.io/v2
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+...
+  meshConfig:
+    extensionProviders:
+      - name: tempo
+        zipkin:
+          service: tempo-sample-distributor.tracing-system.svc.cluster.local
+          port: 9411
+  tracing:
+    sampling: 10000
+    type: None <1>
+  version: v2.5
+---
+kind: ServiceMeshMemberRoll
+apiVersion: maistra.io/v1
+metadata:
+  name: default
+  namespace: istio-system
+spec:
+  members:
+    - tracing-system
+----
+<1> The `spec.tracing.type` setting defines a Distributed Tracing deprecated Jaeger instance. Set `spec.tracing.type` to `None` when connecting to a TempoStack using an `extensionProvider` setting.
++
+[NOTE]
+====
+Create a TempoStack instance _after_ creating the `ServiceMeshControlPlane` and the `ServiceMeshMemberRoll` resources.
+====
++
```

Comment:
```suggestion

```

Reply:
I dont think this "+" does anything for you. You only needed it before the NOTE to keep it in alignment with the preceding step.